frontend/assets/css/colors.scss

```diff
@@ -73,6 +74,7 @@
 
   --button-secondary-color: var(--light-black);
   --button-secondary-hover-color: var(--light-grey-6);
+  --button-secondary-background-color--hover: var(--light-grey-4 );
```

Comment:
due to other Tickets there are two now...imo this should be `cleaned up` in a `separate ticket`, as we should align our naming with @ddog-bitfly

Reply:
@benji-bitfly could you create a ticket and if we should talk about that first invite @ddog-bitfly to our `front end meeting`? 🙏